src/eom-window.c

```diff
@@ -3331,7 +3331,7 @@ show_move_to_trash_confirm_dialog (EomWindow *window, GList *images, gboolean ca
 	} else {
 		if (can_trash) {
 			prompt = g_strdup_printf (ngettext("Are you sure you want to move\n"
-							   "the selected image to the trash?",
+							   "%d selected image to the trash?",
```

Comment:
Why removing the `the` ?

Reply:
I think "the" isn't used with numbers...

Reply:
In my language there is a different meaning with using `the 5` or not.
```
Are you sure you want to move the 5 selected image to the trash?
Möchten Sie die 5 ausgewählten Bilder wirklich in den Papierkorb verschieben?
```
This means exactly those 5 chosen images and not another collection of 5 images.
```
Are you sure you want to move 5 selected image to the trash?
Möchten Sie 5 ausgewählte Bilder wirklich in den Papierkorb verschieben?
```
But this means that 5 images are chosen but it doesn't have to be exactly this collection. It can be another collection of 5 images.
I hope the different is clear.
Not sure if this is a different in origin C language.

Also 2 lines above `the` is used.
```
		if (can_trash) {
			prompt = g_strdup_printf (ngettext("Are you sure you want to move\n"
							   "%d selected image to the trash?",
							   "Are you sure you want to move\n"
							   "the %d selected images to the trash?", n_images), n_images);
```

Reply:
Hmm... ok, I'll leave it in place for now, it can be revised later

Reply:
Thanks.